src/mlpack/methods/ann/ffn.hpp

```diff
@@ -219,6 +248,12 @@ class FFN
    */
   void ResetGradients(arma::mat& gradient);
 
+  /**
+   * Swap the content of this network with given network.
+   * @param network Desired source network.
```

Comment:
I know it is a pedantic request, but could you add newline between the description and the function parameters?